updatecli/updatecli.d/versions.yml

```diff
@@ -0,0 +1,485 @@
+---
+name: Bump release versions in the config/versions.yml
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GITHUB_ACTOR" }}'
+      username: '{{ requiredEnv "GITHUB_ACTOR" }}'
+      owner: '{{ .scm.owner }}'
+      repository: '{{ .scm.repository }}'
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      commitusingapi: true
+      branch: '{{ .scm.branch }}'
+      force: false
+
+actions:
+  elastic-agent:
+    kind: github/pullrequest
+    scmid: githubConfig
+    spec:
+      automerge: false
+      labels:
+        - dependencies
+      title: '[Automation] Bump EDOT versions'
+
+sources:
+  latest-stack-version:
+    name: Get latest Elastic Stack version
+    kind: json
+    spec:
+      file: https://artifacts.elastic.co/releases/stack.json
+      key: "releases[-1].version"
```

Comment:
@v1v @reakaleek Added a stack source using the JSON. The `[-1]` should always grab the last version in the array if I'm not mistaken.